src/sinks/prometheus.rs

```diff
@@ -186,11 +186,16 @@ impl Sink for PrometheusSink {
         self.start_server_if_needed();
 
         match event.into_metric() {
-            Metric::Counter { name, val } => self.with_counter(name, |counter| counter.inc_by(val)),
+            Metric::Counter {
+                name,
+                val,
+                timestamp: _,
```

Comment:
We need to doc that this sink and statsd do not support event provided timestamps.

Reply:
I'm uncertain what's the best way of doing it. @binarylogic calling for help!

Reply:
Done, I've updated the docs.